config/go.d/prometheus.conf

```diff
@@ -0,0 +1,160 @@
+# netdata go.d.plugin configuration for prometheus
+#
+# This file is in YaML format. Generally the format is:
+#
+# name: value
+#
+# There are 2 sections:
+#  - GLOBAL
+#  - JOBS
+#
+#
+# [ GLOBAL ]
+# These variables set the defaults for all JOBs, however each JOB may define its own, overriding the defaults.
+#
+# The GLOBAL section format:
+# param1: value1
+# param2: value2
+#
+# Currently supported global parameters:
+#  - update_every
+#    Data collection frequency in seconds. Default: 1.
```

Comment:
Let's not make the default `1s` because hitting a `/metrics` endpoint that hard will be a) unnecessary HTTP traffic and load and b) noisey in server logs. Good default: `5s`

Reply:
Agree, in fact i made it `5s` by default

https://github.com/ilyam8/go.d.plugin/blob/46e45d32bc84f4193f5a40f73f4a723276b4cbd6/modules/prometheus/prometheus.go#L12-L20

Forgot to change it in the config